src/CMakeLists.txt

```diff
@@ -757,16 +760,19 @@ endif()
 
 rocm_install_targets(
   TARGETS MIOpen
-  INCLUDE
-    ${PROJECT_SOURCE_DIR}/include
-    ${PROJECT_BINARY_DIR}/include
-    ${MIOPEN_CXX_HEADER_PATH}
-  PREFIX ${MIOPEN_INSTALL_DIR}
```

Comment:
This should not be removed. Headers and targets should be installed with `rocm_install_targets`.